cdap-distributions/src/emr/install.sh

```diff
@@ -110,28 +110,22 @@ if [ -n "${CDAP_TAG}" ]; then
 fi
 
 # Setup cookbook repo
-test -d /var/chef/cookbooks && rm -rf /var/chef/cookbooks
-${__packerdir}/cookbook-dir.sh || die "Failed to setup cookbook dir"
+test -d /var/chef/cookbooks && sudo rm -rf /var/chef/cookbooks
+sudo ${__packerdir}/cookbook-dir.sh || die "Failed to setup cookbook dir"
 
 # Install cookbooks via knife
-${__packerdir}/cookbook-setup.sh || die "Failed to install cookbooks"
+sudo ${__packerdir}/cookbook-setup.sh || die "Failed to install cookbooks"
 
 # CDAP cli install, ensures package dependencies are present
 # We must specify the cdap version
 echo "{\"cdap\": {\"version\": \"${CDAP_VERSION}\"}}" > ${__tmpdir}/cli-conf.json
-chef-solo -o 'recipe[cdap::cli]' -j ${__tmpdir}/cli-conf.json
+sudo chef-solo -o 'recipe[cdap::cli]' -j ${__tmpdir}/cli-conf.json
```

Comment:
probably should add a ``|| die ...`` here.  (and i need to update the HDInsight script also)